core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -119,6 +119,8 @@ public class RemoteLogManager implements Closeable {
     private static final Logger LOGGER = LoggerFactory.getLogger(RemoteLogManager.class);
     private static final String REMOTE_LOG_READER_THREAD_NAME_PREFIX = "remote-log-reader";
 
+    private static final Set<Uuid> deletedTopicIds = ConcurrentHashMap.newKeySet();
```

Comment:
It's weird to have this field as a static member. Could we make it an instance member?